### ISSUE TITLE
Using an environment variable in config

### ISSUE THREAD

**Issue by amireldor** [lektor/lektor#511]
Can I use an environment variable in a config file? For example, `configs/disqus-comments.ini`?

```
$ cat configs/disqus-comments.ini
shortname = "$MY_AWESOME_DISQUS_SHORTNAME"
```
I am thinking of having my site's code public and think keeping this stuff private is essential. Databags might be a direction but I'm new and unsure of anything.

Forgive me if duplicate, couldn't find info.

**Comment by goanpeca** [comment]
I don't think this is possible at the moment, but I don't see why in this case this is needed, as anyone can explore the HTML and see the shortname. There is no real protection anyway so why bother?

**Comment by amireldor** [comment]
Hmm. Right. That makes sense. Heh.

I'm trying to think how env vars might be needed or used but I think 'themes' are a solution when sharing a reusable part for a site.

Plugins are pure Python right? So that can access the env vars any time.

**Comment by goanpeca** [comment]
>Plugins are pure Python right? So that can access the env vars any time.

Correct :-)

**Comment by roadsideseb** [comment]
I've put together a plugin that makes environment variables available inside of templates: https://pypi.org/project/lektor-envvars. Maybe that's useful for some scenarios.
